doc/source/reference/random/brng/xorshift1024.rst

```diff
@@ -0,0 +1,35 @@
+Xorshift1024*φ
```

Comment:
This generator should also probably be dropped.  It fails statistical tests for quality.

Reply:
I'll wait to hear back whether this, and the other two, should be removed.  Probably a good idea to start with a smaller API since easier to expand than contract.  This can always go in a stand along package of bit-generators along with PCG32/64.